hq/markdown/parameter-store.md

```diff
@@ -0,0 +1,57 @@
+# SSM Parameters
```

Comment:
If this is documenting the Guardian wrapper library, can we just point people at its README?